types/xmpp__client/dist/xmpp.min.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from './xmpp';
```

Comment:
what's the`xmpp.min`? It's nt a module,  it's just minified umd, isn't it? How can TSC resolve into that file, so `xmpp.min.d.ts` is required?

Reply:
Well, it's still UMD, so if anyone imports it from TS, it will work like any other module.

Reply:
How can I import 'xmpp.min' from TS? (so by extension in JS)

Reply:
`import * as xmpp from '@xmpp/client/dist/xmpp.min'`

Reply:
cool, sorry to be a pita :)